libr/core/disasm.c

```diff
@@ -3814,7 +3814,6 @@ static bool ds_print_core_vmode(RDisasmState *ds, int pos) {
 	case R_ANAL_OP_TYPE_RJMP:
 	case R_ANAL_OP_TYPE_RCALL:
 		if (ds->analop.jump != UT64_MAX && ds->analop.jump != UT32_MAX) {
-			ds->analop.jump = get_ptr_ble (ds, ds->analop.jump);
```

Comment:
uhm, RCALL/RJMP, is about register reference, the `get_ptr_ble` should be only for the MJMP, MCALL, .. so i think your patch is good

Reply:
i think the check in 				`if (ds->asm_hint_call_indirect) {` is covering that for the UCALL/UJMP, maybe we should handle MJMP or just remove it and make it UJMP, but U is about unknown.. and there are probably more places where MJMP is misused, can you check that too?

Reply:
will be good to have some tests for x86-64 and arm64 on this

Reply:
diff between `call rax`, `call [rax]`, `call [0x80080]` ,..